website/sidebars.json

```diff
@@ -94,6 +94,7 @@
         "items": [
           "the-new-architecture/pillars",
           "the-new-architecture/pillars-turbomodules",
+          "the-new-architecture/pillars-cxxturbomodules",
           "the-new-architecture/pillars-fabric-components",
           "the-new-architecture/pillars-codegen"
         ]
```

Comment:
As suggested previously, I would probably:
1. add a new category below the Pillars one
2. Call it Advanced
3. Add a page for the `pillars-cxxturbomodules`
4. Add a page for the `extending cxx type` page (to export from the TM guide).
I think this will make the website easier to navigate and to consume for the user.

What do you think?

Reply:
Solved with comment ^ https://github.com/facebook/react-native-website/pull/3439#discussion_r1045624871